### PR TITLE
[WFSSL-107] Add a note in the README about the OpenSSL versions that wildfly-openssl is compatible with

### PR DESCRIPTION
**Pull request by sudeshnas93**
Issue: https://issues.redhat.com/browse/WFSSL-107

### CODE REVIEW

**Comment by sudeshnas93** [comment]
Hello @fjuma , can you please review and merge?

File: README.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ SSLContext should be usable as a drop in replacement for applications that are c
 This code was originally based on the Tomcat Native code, however it has been fairly extensively modified to more closely
 align with JSSE and to support dynamic linking.
 
+wildfly-openssl is compatible with both the OpenSSL 3.0 series and the OpenSSL 1.1.1 series.
```
Comment (fjuma) [inline README.md:10]: Just something very minor, instead of "wildfly-openssl is...", let's use "WildFly OpenSSL is..."

**Review by fjuma — CHANGES_REQUESTED** [review]
Please squash the commits and then this should be ready to go.

**Review by fjuma — APPROVED** [review]
Thanks @sudeshnas93!

**Pull request merged**